src/Actor.cpp

```diff
@@ -279,24 +281,56 @@ void Actor::LoadFromNode( const XNode* pNode )
 	PlayCommandNoRecurse( Message("Init") );
 }
 
+bool Actor::PartiallyOpaque()
+{
+	return m_pTempState->diffuse[0].a > 0 || m_pTempState->diffuse[1].a > 0 ||
```

Comment:
I'm not sure how to respond to this, I made the PartiallyOpaque function to be a reusable check with its purpose stated by its name, instead of the long condition with a comment that was used before.